addons/web/static/lib/popper/popper.js

```diff
@@ -1238,7 +1238,7 @@ function computeStyle(data, options) {
   // values, while left/right prefers floored values.
   var offsets = {
     left: Math.floor(popper.left),
-    top: Math.round(popper.top),
+    top: Math.max(Math.round(popper.top), 0),
```

Comment:
Change in the lib file, you are not supposed to be allowed to do that... is this actually needed? If yes:

1) Check if the updated version of library has not already fixed this issue, maybe we must simply update it
2) Try to patch the library from an Odoo file, not in the library file directly. We have a folder "libs" in the web app where we monkey patch the libs for instance.